Dockerfile

```diff
@@ -64,6 +64,12 @@ RUN mkdir -p /go/src/github.com/sql-machine-learning && \
     go install -v ./... && \
     cd /
 
+# Install latest sqlflow_models for testing custom models, see main_test.go:CaseTrainCustomModel
+RUN git clone https://github.com/sql-machine-learning/models.git && \
```

Comment:
Should we make a git submodule in the sqlflow repo pointing to the models repo?

Reply:
I could be wrong but submodule I thought should be a **required** dependency, but in this case, SQLFlow can still work if `sqlflow_models` is not installed.

Reply:
Good point. I take back my suggestion.